### PR TITLE
Update contracts, Ethereum/IPFS configs

### PR DESCRIPTION
**Pull request by skddc**
* Use our new IPFS node
* Switch from our own Parity to Infura in prod
* Use contracts from npm

### CODE REVIEW

**Comment by bumi** [comment]
looks good. 
did you test it without metamask? I have #61 open which is kinda related. 

but in anycase this looks good and I will merge it.

File: config/environment.js [hunk 1]
```diff
@@ -52,8 +52,8 @@ module.exports = function(environment) {
     // ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.contractMetadata['networkId'] = '100';
-    ENV.web3ProviderUrl = 'http://localhost:7545';
+    ENV.contractMetadata['networkId'] = '42';
```
Comment (bumi) [inline config/environment.js]: actually I am not sure about this one. by default we run the local ganache network as network ID 100. 
This change requires to always provide that network ID for development with a local network. 
maybe we can add some npm scripts for that? 
`npm run local` `npm run kovan` or something similar?

Comment (skddc) [reply config/environment.js]: The reason for this is that the default is to not develop contracts, when developing the Web UI, so one shouldn't have to deal with ganache and have to use minimal seed data by default. Scripts would be nice, but I don't think it should be part of the same PR.

**Comment by skddc** [comment]
> did you test it without metamask? I have #61 open which is kinda related.

Things are quite broken for me in both master and this one, when not logged in to a metamask account. All our init and loading stuff has been broken since I last wrote any code for kredits-web. So it's also broken in this PR/branch.

**Comment by bumi** [comment]
actually it seems to work well for me with: 
`NETWORK_ID=100 WEB3_PROVIDER_URL=http://localhost:7545 ember s`

running the app without metamask using my local network

**Comment by skddc** [comment]
So it's broken when using a real node. Maybe that helps with debugging.

**Comment by bumi** [comment]
you mean no metamask and default settings meaning kovan network and contract addresses from the kredits-contracts package?

**Pull request merged**